### DISCUSSION TITLE
Command+Click does not work.

### DISCUSSION THREAD

**Discussion by Atom2004** [rustdesk/rustdesk#10189] [General]
### Bug Description

I set a keymap of Command + Click for jump to declaration in such as Intellij and VSCode. But it does not work over RustDesk.

When I press Command, the code that the mouse hovers changes with some visual effects. But when I press the mouse button, intellij doesn't implement the jump. Please refer to Screen Recording for details.

I try to connect the same keyboard and mouse into MacOS via USB port. It works.

The same behavior in vscode.

### How to Reproduce

1. Install intellij or vscode in macos.
2. Set the keymap of Command + Click.
3. Remote connect from windows 11 via RustDesk.
4. Try Command + Click in some code project.

### Expected Behavior

Intellij should jump the code to the declaration of mouse click.

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 11 -> macOS

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.4 -> 1.3.4

### Screenshots

Screen shot of the keymap setting in Intellij.
![image](https://github.com/user-attachments/assets/dbd80940-f6fc-4f3d-aee0-77793ca0a4a7)

Screen Recoding. Key recoding and mouse button recoding under bottom right corner.
https://github.com/user-attachments/assets/def085c0-cdb1-41c7-84e9-e9724707f4e4

**Comment by web-xiaxia** [comment]
1.4.4 still the same
